### DISCUSSION TITLE
Owner rights on placed down coasters/sceanery/etc

### DISCUSSION THREAD

**Discussion by Nubbie** [OpenRCT2/OpenRCT2#15700] [Feature requests]
Some games gives the player a specific ID (in this case it could be our 'hash') and only let the person who have built it, touch it (great for greifers who delete/close/remove rides)

The only way for others to touch that coaster/store would be if they accept a permission for that player to make edits on the thing the original creator placed down

**Comment by janisozaur** [comment]
Not possible until we get our own format where we can store such things.

**Comment by IntelOrca** [comment]
Until you get greifers who obstruct other players with rides.

**Comment by Nubbie** [comment]
@intelorca could be done now too - if the other player doesn't have deletion rights, it would be in the way

If the player doesn't have the rights, he could use edit that coaster to remove pieces from it, which the greifer could do with other peoples coasters

(Common for greifers who don't have 'remove ride' permission is to edit the ride and remove every piece until nothing is left)
